src/parse.rs

```diff
@@ -48,9 +48,12 @@ pub enum Tag<'a> {
     Heading(u32),
 
     BlockQuote,
-    /// A code block. The value contained in the tag describes the language of the code,
-    /// which may be empty.
-    CodeBlock(CowStr<'a>),
+    /// A code block.
+    ///
+    /// The boolean is `true` is this is an indented code block (not starting with "\`\`\`").
```

Comment:
```suggestion
    /// The boolean is `true` if this is an indented code block (not starting with ```` ``` ```` or `~~~`).
```
Rendered (with GitHub Markdown): /// The boolean is `true` if this is an indented code block (not starting with ```` ``` ```` or `~~~`).